var promise = Promise.reject(new Error('REJECTED!')).catch(err => console.log(err.message));